core/src/main/java/org/elasticsearch/env/Environment.java

```diff
@@ -43,6 +47,15 @@
 // TODO: move PathUtils to be package-private here instead of
 // public+forbidden api!
 public class Environment {
+    public static final Setting<String> PATH_HOME_SETTING = new Setting<>("path.home", "", Function.identity(), false, Setting.Scope.CLUSTER);
```

Comment:
if you want to add a `Setting.simpleString(String key, boolean dynamic, Scope scope)` method I am ok with that?